src/config/hashing.rs

```diff
@@ -136,9 +136,9 @@ where
     }
 
     #[must_use]
-    pub fn get_or_init(&self, f: impl FnOnce() -> T) -> Option<T> {
-        let value = f();
+    pub fn get_or_init(&self, f: impl FnOnce() -> T) -> Option<&'static T> {
```

Comment:
`get_or_init` alreays return `Some` so it should be simply `&'static T`.

Reply:
This can still be null since the AHASH_SEED envvar isnt always set

Reply:
Well, `T` itself is `Option`, so if `AHASH_SEED` isn't set, it'll return `&'static None`.

Reply:
No worries, I'll just merge it and fiddle with it later.